src/tools/tools/Logger.fs

```diff
@@ -82,6 +83,9 @@ type ILoggerProvider =
     /// Takes three parameters: log id, log level, log Message
     abstract member Log : (string*LogLevel*string) -> unit
 
+    /// Takes four parameters: log id, log level, JobID, log Message
+    abstract member Log : (string*Guid*LogLevel*string) -> unit
+
```

Comment:
To be consistent, should have an overload that  does (Guid * LogLevel * String) -> unit